packages/audio-native/scripts/build.mjs

```
@@ -0,0 +1,27 @@
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+const isWindows = platform() === "win32";
+const addonPath = join(__dirname, "..", "build", "Release", "audio_native.node");
+
+if (isWindows) {
+  console.log("Building native addon (node-gyp rebuild)...");
+  try {
+    execSync("node-gyp rebuild", { stdio: "inherit" });
+  } catch {
+    if (existsSync(addonPath)) {
+      console.warn("node-gyp rebuild failed (file may be locked by a running process). Using existing native addon.");
```

ESLint

[error] 'console' is not defined.

(no-undef)